site/content/en/docs/Tasks/configmaps_and_secrets.md

```diff
@@ -1,8 +1,51 @@
 ---
 title: "ConfigMaps and Secrets"
 linkTitle: "ConfigMaps and Secrets"
-weight: 3
+weight: 1
 date: 2017-01-05
 description: >
   Working with ConfigMaps and Secrets
----
\ No newline at end of file
+---
+
+{{< alert color="success" title="TL;DR" >}}
+- Generate ConfigMaps from files and literals with `configMapGenerator`
+- Generate Secrets from files and literals with `secretGenerator`
+- Rollout ConfigMap and Secret changes to workloads
+{{< /alert >}}
+
+## Motivation
```

Comment:
I don't think the section name **Motivation** is necessary.

```suggestion

```

Reply:
The **Motivation** header was copied from the original site [here](https://kubectl.docs.kubernetes.io/guides/config_management/secrets_configmaps/). All other Task pages include a Motivation section as an introduction to the concept.

Personally, I think it makes sense to keep this header but I can remove if you feel strongly that it is not helpful.

Reply:
I think it's OK to keep the header if you think it's useful.